packages/magento-customer/components/ChangePasswordForm/ChangePasswordForm.tsx

```diff
@@ -22,49 +22,53 @@ export function ChangePasswordForm() {
     ChangePasswordMutation,
     ChangePasswordMutationVariables & { confirmPassword?: string }
   >(ChangePasswordDocument)
-  const { muiRegister, handleSubmit, required, watch, data, formState, error } = form
+  const { handleSubmit, required, data, formState, error, control } = form
+  const [remainingError, inputError] = graphqlErrorByCategory({
+    category: 'graphql-input',
+    error,
+  })
+
   const submitHandler = handleSubmit(() => {})
-  const pass = watch('newPassword')
 
   return (
     <Form onSubmit={submitHandler} noValidate>
       <FormRow>
-        <TextField
+        <PasswordElement
+          control={control}
+          name='currentPassword'
           variant='outlined'
           type='password'
-          error={!!formState.errors.currentPassword}
           label={<Trans id='Current Password' />}
+          error={!!inputError}
           required={required.currentPassword}
-          {...muiRegister('currentPassword', { required: required.currentPassword })}
-          helperText={formState.errors.currentPassword?.message}
           disabled={formState.isSubmitting}
+          helperText={formState.errors.currentPassword?.message || inputError?.message}
```

Comment:
I'm not sure if this is necessary in combination with PasswordElement, you might be able to omit this?